libqtile/widget/battery.py

```diff
@@ -205,6 +212,8 @@ def _get_text(self):
             elif info['stat'] == CHARGING:
                 char = self.charge_char
                 time = (info['full'] - info['now']) / info['power']
+            elif info['stat'] == UNKNOWN:
```

Comment:
Why do you do this? I think usually an `UNKNOWN` status means fully charged and plugged into AC power. Wouldn't this break the formatting on the final return (since `char` isn't defined)?